### PR TITLE
Update iq.m3u

### PR DESCRIPTION
**Pull request by Redo-San**
I added Al Rabiaa TV channel and modified the link of Dijlah TV channel

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/iq.m3u [hunk 1]
```diff
@@ -33,10 +35,8 @@ https://5d94523502c2d.streamlock.net/alsharqiyalive/mystream/playlist.m3u8
 https://media.streambrothers.com:1936/8248/8248/playlist.m3u8
 #EXTINF:-1 tvg-id="AssyrianANB.iq",Assyrian ANB (720p)
 https://anbsat.ddns.net/anb/live/ANB/playlist.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-http://91.134.145.75:10001/Dijlah/index.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-https://ghaasiflu.online/Dijlah/index.m3u8
+#EXTINF:-1 tvg-id="dijlah.tv",Dijlah TV 
+https://ghaasiflu.online/Dijlah/tracks-v1a1/mono.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:39]:
```suggestion
https://ghaasiflu.online/Dijlah/index.m3u8
```

File: streams/iq.m3u [hunk 2]
```diff
@@ -33,10 +35,8 @@ https://5d94523502c2d.streamlock.net/alsharqiyalive/mystream/playlist.m3u8
 https://media.streambrothers.com:1936/8248/8248/playlist.m3u8
 #EXTINF:-1 tvg-id="AssyrianANB.iq",Assyrian ANB (720p)
 https://anbsat.ddns.net/anb/live/ANB/playlist.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-http://91.134.145.75:10001/Dijlah/index.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:37]: works

File: streams/iq.m3u [hunk 3]
```diff
@@ -33,10 +35,8 @@ https://5d94523502c2d.streamlock.net/alsharqiyalive/mystream/playlist.m3u8
...
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-https://ghaasiflu.online/Dijlah/index.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:39]: works

**Comment by UltraHDR** [comment]
Thanks @Redo-San for your contribution. The existing Dijlah links work and the Al Rabiaa channel URL updates every few hours and cannot be added. Please re-open if you find a non expiring Al Rabiaa link.

**Pull request closed without merging**